.github/workflows/unittests.yml

```
@@ -22,12 +22,21 @@ jobs:
         with:
           python-version: ${{ matrix.python }}
           channels: conda-forge
-      - run: conda --version
-      - run: which python
-      - run: ./ci/install_mpi.sh
+      - name: Install MPI
+        run: |
+          conda --version
+          which python
+          ./ci/install_mpi.sh
         env:
           MPI_INSTALL: ${{ matrix.mpi_install }}
-      - run: pip install pytest pytest-env
-      - run: pip install -e .
-      - run: echo "localhost slots=16">hostfile
-      - run: pytest
+      - name: Run unit tests
+        run: |
+          pip install -e .[test]
+          echo "localhost slots=16">hostfile
+          pytest --cov-report=xml --cov=dicodile
+      - name: Upload coverage
+        uses: codecov/codecov-action@v1
+        with:
+          flags: unittests 
+          fail_ci_if_error: true 
+          verbose: true
```

Missingn an extra line?
```suggestion
          verbose: true

```

---

Fixed, thanks!